### ISSUE TITLE
TypeError 'UrlInfo' object is not iterable error when loading oxford_iiit_pet dataset.

### ISSUE THREAD

**Issue by alvarobasi** [tensorflow/datasets#2024]
**Short description**
Exception has occurred: TypeError
'UrlInfo' object is not iterable error is triggered when trying to load a TF dataset.

I've tried with other datasets such as dogs_vs_cats with the same result.
**Environment information**
* Operating System: Ubuntu 18.04 LTS
* Python version: 3.6.9
* `tfds-nightly` version: 3.1.0.dev202005130105
* `tf-nightly-gpu` version: 2.2.0.dev20200508

**Reproduction instructions**

```
(train_dataset, test_dataset), info = tfds.load(name="oxford_iiit_pet:3.*.*",
                                                split=["train", "test"],
                                                shuffle_files=True,
                                                as_supervised=True,
                                                with_info=True)
```

**Link to logs**
If applicable, <link to gist with logs, stack trace>

**Expected behavior**
What you expected to happen.

**Additional context**

**Comment by vijayphoenix** [comment]
Thanks for reporting. It should be fixed by https://github.com/tensorflow/datasets/pull/2022

**Comment by vijayphoenix** [comment]
Fixed by f096c13.
Can be closed
